tests/integration/test_mean.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2021-2022 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import numpy as np
+import pytest
+
+import cunumeric as num
+
+DIM = 7
+
+NO_EMPTY_SIZE = (
+    (1,),
+    (DIM,),
+    (1, 1),
+    (1, DIM),
+    (DIM, 1),
+    (DIM, DIM),
+    (1, 1, 1),
+    (DIM, 1, 1),
+    (1, DIM, 1),
+    (1, 1, DIM),
+    (DIM, DIM, DIM),
+)
+
+
+def gen_out_shape(size, axis):
+    if axis is None:
+        return ()
+    if axis < 0:
+        axis += len(size)
+    if axis >= 0 and axis < len(size):
+        return size[:axis] + size[axis + 1 :]
+    else:
+        return -1
+
+
+@pytest.mark.parametrize("arr", ([], [[], []]))
+def test_empty_arr(arr):
+    res_np = np.mean(arr)
+    res_num = num.mean(arr)
+    assert np.isnan(res_np) and np.isnan(res_num)
+
+
+def test_scalar():
+    res_np = np.mean(10)
+    res_num = num.mean(10)
```

Comment:
suggest parametrize, to cover negative, floating point, complex, zero value scalar cases, etc.

Reply:
Fixed. Thanks.